sdk/core/az_core/inc/az_http_transport.h

```diff
@@ -168,6 +168,44 @@ AZ_NODISCARD int32_t az_http_request_headers_count(_az_http_request const* reque
 AZ_NODISCARD az_result
 az_http_client_send_request(_az_http_request* p_request, az_http_response* p_response);
 
+/**
+ * @brief Declaring az_http_policy for using it to create policy process callback
+ * _az_http_policy_process_fn definition. Definition is added below after it.
+ *
+ */
+typedef struct _az_http_policy _az_http_policy;
+
+/**
+ * @brief Defines the callback signature of a policy process which should receive an
+ * _az_http_policy, options reference (as void *), an _az_http_request and az_http_response.
+ *
+ * void * is used as polymorphic solution for any policy. Each policy implementation would know the
+ * specif pointer type to cast options to.
+ *
+ */
+typedef AZ_NODISCARD az_result (*_az_http_policy_process_fn)(
```

Comment:
Move this higher in this .h file

Reply:
I moved it as high in this file, as possible.

Reply:
Done.